integrators/embedded_explicit_generalized_runge_kutta_nyström_integrator_body.hpp

```diff
@@ -374,7 +373,7 @@ internal::EmbeddedExplicitGeneralizedRungeKuttaNyströmIntegrator<Method,
                                                                  ODE_> const&
 EmbeddedExplicitGeneralizedRungeKuttaNyströmIntegrator() {
   static_assert(
-      std::is_base_of<methods::EmbeddedExplicitGeneralizedRungeKuttaNyström,
+      std::is_base_of<_methods::EmbeddedExplicitGeneralizedRungeKuttaNyström,
```

Comment:
Keep the using directive, drop the `_methods` here.

Reply:
No can do, at least not in this PR.

Note that the using directive that I removed was in the `internal` namespace, but here we are no longer in the `internal` namespace, we are in `_embedded_explicit_generalized_runge_kutta_nyström_integrator`.  If I put a using directive of `_methods` here, any client of the integrator will have direct visibility on *all* the methods.  We don't want that.

The proper fix is to move the implementation of this factory method into `internal`, where it would benefit from the using directives without creating leaks.  Unfortunately that's not that simple because the method has the same name as the class.  So it will probably have to go into `internal::factory` or some such to avoid the name clash.

A useful cleanup, but what do we say to the God of Feature Creep?  Not today.